osu-winello.sh

```diff
@@ -102,33 +102,37 @@ Install()
       
       fi
     fi
-osid=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
-if command -v pacman >/dev/null 2>&1 && [ "$osid" == "steamos" ]; then
-    Info "SteamOS detected, installing dependencies..."
-    Info "Please enter your password when asked"
-    Info "Installing packages and wine-staging dependencies.."
-    if command -v wine >/dev/null 2>&1 ; then
-        Info "Wine (possibly) already found, removing it to replace with staging.."
-        sudo pacman -Rdd --noconfirm wine || Info "Looks like staging is already installed"
+      osid=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
+
+      if [ "$osid" != "steamos" ] ; then
+
+        if ! grep -q -E '^\[multilib\]' '/etc/pacman.conf'; then
+          Info "Enabling multilib.."
+          printf "\n# Multilib repo enabled by osu-winello\n[multilib]\nInclude = /etc/pacman.d/mirrorlist\n" | "$root_var" tee -a /etc/pacman.conf
+        fi
+
+        Info "Installing packages and wine-staging dependencies.."
+        if command -v wine >/dev/null 2>&1 ; then
+          Info "Wine (possibly) already found, removing it to replace with staging.."
+          "$root_var" pacman -Rdd --noconfirm wine || Info "Looks like staging is already installed"
+        
+        fi
+          "$root_var" pacman -Sy --noconfirm --needed git base-devel p7zip wget zenity wine-staging winetricks giflib lib32-giflib libpng lib32-libpng libldap lib32-libldap gnutls lib32-gnutls mpg123 lib32-mpg123 openal lib32-openal v4l-utils lib32-v4l-utils libpulse lib32-libpulse alsa-plugins lib32-alsa-plugins alsa-lib lib32-alsa-lib libjpeg-turbo lib32-libjpeg-turbo libxcomposite lib32-libxcomposite libxinerama lib32-libxinerama ncurses lib32-ncurses opencl-icd-loader lib32-opencl-icd-loader libxslt lib32-libxslt libva lib32-libva gtk3 lib32-gtk3 gst-plugins-base-libs lib32-gst-plugins-base-libs vulkan-icd-loader lib32-vulkan-icd-loader cups samba dosbox || Error "Some libraries didn't install for some reason, check pacman or your connection"
+          Info "Dependencies done, skipping.." 
+      
+      else
+      check=$(sudo touch -c /usr/ 2>&1)
+    if echo "$check" | grep -q "touch: setting times of '/usr/': Read-only file system"; then
+      echo -e '\033[1;31m'"The Steam Deck's file system is in read-only mode, preventing further action. To continue, you must disable read-only mode. More information can be found on GitHub.\033[0m"
+    exit 1
+  else
```

Comment:
There's no need for this else here. I mean, if it's working, why would you tell the user file-system is fine? Just let it go xd. You can just close with a simple `fi`.

Reply:
Also no need to use `echo` when you have the script's built-in functions, which also reverts install in case it fails.
Just go with:

`Error "The Steam Deck's file system is in read-only mode, preventing further action. To continue, you must disable read-only mode. More information can be found on GitHub."`

Reply:
> Also no need to use `echo` when you have the script's built-in functions, which also reverts install in case it fails. Just go with:
> 
> `Error "The Steam Deck's file system is in read-only mode, preventing further action. To continue, you must disable read-only mode. More information can be found on GitHub."`

Could you please look into this too? Also please don't change the steamos check part, that'd be confusing for me too xd
When you're done with all changes, just do another PR with everything corrected (or basically, with new files).